packages/thirdweb/src/transaction/serialize-transaction.test.ts

```diff
@@ -24,29 +16,31 @@ const BASE_TRANSACTION = {
   to: checksumAddress(TEST_ACCOUNT_B.address),
   nonce: 785,
   value: toWei("1"),
-} as const satisfies TransactionSerializableBase;
+} as const;
 
 describe.runIf(process.env.TW_SECRET_KEY)("eip1559", () => {
   const baseEip1559 = {
     ...BASE_TRANSACTION,
     chainId: 1,
     maxFeePerGas: fromGwei("2"),
     maxPriorityFeePerGas: fromGwei("2"),
-  } as const satisfies TransactionSerializableEIP1559;
+  } as const;
 
   test("default", () => {
-    const serialized = serializeTransaction<TransactionSerializableEIP1559>({
+    const serialized = serializeTransaction({
       transaction: baseEip1559,
     });
-    assertType<TransactionSerializedEIP1559>(serialized);
     expect(serialized).toEqual(
       "0x02ef0182031184773594008477359400809470997970c51812dc3a010c7d01b50e0d17dc79c8880de0b6b3a764000080c0",
     );
 
-    const tx = parseTransaction(serialized);
+    const tx = ox__TransactionEnvelopeEip1559.deserialize(
+      serialized as ox__TransactionEnvelopeEip1559.Serialized,
```

Comment:
will these cast go away once we migrate everything?

Reply:
These are just the tests being extra picky because I'm allowing legacy transactions